src/azure-cli/azure/cli/command_modules/acr/agentpool.py

```diff
@@ -84,9 +88,9 @@ def acr_agentpool_delete(cmd,
     user_confirmation("Are you sure you want to delete the agentpool '{}' in registry '{}'?".format(
         agent_pool_name, registry_name), yes)
     try:
-        response = client.delete(resource_group_name=resource_group_name,
-                                 registry_name=registry_name,
-                                 agent_pool_name=agent_pool_name)
+        response = client.begin_delete(resource_group_name=resource_group_name,
```

Comment:
same question above about whether this is consistent with old behavior now we are calling "begin_delete"

Reply:
Just name change, no behavior change.